vendor.conf

```diff
@@ -126,10 +126,10 @@ github.com/containerd/containerd                    4d242818bf55542e5d7876ca276f
 github.com/containerd/fifo                          ff969a566b00877c63489baf6e8c35d60af6142c
 github.com/containerd/continuity                    26c1120b8d4107d2471b93ad78ef7ce1fc84c4c4
 github.com/containerd/cgroups                       44306b6a1d46985d916b48b4199f93a378af314f
-github.com/containerd/console                       8375c3424e4d7b114e8a90a4a40c8e1b40d1d4e6
+github.com/containerd/console                       8375c3424e4d7b114e8a90a4a40c8e1b40d1d4e6 # v1.0.0
 github.com/containerd/go-runc                       7016d3ce2328dd2cb1192b2076ebd565c4e8df0c
-github.com/containerd/typeurl                       b45ef1f1f737e10bd45b25b669df25f0da8b9ba0
-github.com/containerd/ttrpc                         0be804eadb152bc3b3c20c5edc314c4633833398
+github.com/containerd/typeurl                       b45ef1f1f737e10bd45b25b669df25f0da8b9ba0 # v1.0.0-13-gb45ef1f
+github.com/containerd/ttrpc                         0be804eadb152bc3b3c20c5edc314c4633833398 # v1.0.0-16-g0be804e
```

Comment:
FWIW; both of these appear to be ahead of what containerd/containerd uses (https://github.com/containerd/containerd/blob/b1f514641f328ad20d0444ad3837947105c76434/vendor.conf#L10-L11);

https://github.com/containerd/ttrpc/compare/6e416eafd26e6e738df716b21d421d5b59702bb4...0be804eadb152bc3b3c20c5edc314c4633833398

Wondering if we should roll them back to the last tagged release, or if we should update containerd and see if a new tag can be created for them (I'll look at that, but in the meantime?)